rel/jenkins/percona-create-repos.groovy

```diff
@@ -0,0 +1,45 @@
+library changelog: false, identifier: 'lib@ENG-866_automate_repos_creation', retriever: modernSCM([
+    $class: 'GitSCMSource',
+    remote: 'https://github.com/vorsel/jenkins-pipelines.git'
+]) _
+
+pipeline {
+    agent {
+        label 'source-builder'
+    }
+    parameters {
+        string(
+            defaultValue: '',
+            description: 'Enter name for the new repo(i.e. psmdb-44)',
+            name: 'REPO_NAME')
+        string(
+            defaultValue: 'release,testing,experimental,laboratory',
+            description: 'Repo components to create(coma separated)',
+            name: 'COMPONENTS')
+    }
```

Comment:
we need more params such as LIMITS and OS

Reply:
Done